mandarzope/magento2
# CMS Page provider returns result with 'id'

### Description (*)
CMS Page provider results contained page_id instead of id 

### Fixed Issues (if relevant)
1. Fixes magento/magento2#34908

### Manual testing scenarios (*)
1. Install Magento 2.4 and install sample data
2. execute GraphQL query `query {  route(url: "home") { __typename relative_url redirect_code } }`
3. X-Magento-Tags response header should contain 'cms_p,cms_p_2'

### Questions or comments

### Contribution checklist (*)
 - [ *] Pull request has a meaningful description of its purpose
 - [* ] All commits are accompanied by meaningful commit messages
